metacat-metadata-mysql/src/test/groovy/com/netflix/metacat/metadata/mysql/BaseSpec.groovy

```diff
@@ -34,13 +45,32 @@ import java.util.concurrent.atomic.AtomicBoolean
 import static java.lang.String.format
 
 @Ignore
```

Comment:
Does this still mean to be `@Ignore`?

Reply:
This was added to not run the methods of this class.